### PR TITLE
Add sudoers permission to docker user

### PR DESCRIPTION
**Pull request by lxn2**
This gives the docker user sudoers permissions.

### CODE REVIEW

**Comment by lxn2** [comment]
Can we merge this so @kevinthesun can reconfigure his nightly jobs to clean up workspaces?

File: tests/jenkins/run_as_user.sh [hunk 1]
```diff
@@ -18,4 +18,5 @@ groupadd -f -g ${GROUP_ID} ${GROUP_NAME}
 useradd -m -u ${USER_ID} -g ${GROUP_NAME} ${USER_NAME}
 chown -R ${USER_NAME}:${GROUP_NAME} ${HOME_DIR}
 chown -R ${USER_NAME}:${GROUP_NAME} /usr/local/lib/
+echo "%${GROUP_NAME}  ALL=(ALL)       NOPASSWD: ALL" >> /etc/sudoers
```
Comment (piiswrong) [inline tests/jenkins/run_as_user.sh]: /etc/sudoers will get longer each run. manually add this in?

Comment (lxn2) [inline tests/jenkins/run_as_user.sh]: This script should only be called inside a docker container, so this is /etc/sudoers file of the container.

**Comment by lxn2** [comment]
ping @piiswrong

**Pull request merged**